src/App.css

```diff
@@ -0,0 +1,13 @@
+html,
+body {
+  margin: 0;
+  height: 100%;
+}
+#root {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
```

Comment:
This line is not needed. I checked out the branch and confirmed that the content of the page is higher than that.

Reply:
https://alexcarpenter.me/posts/2018/09/sticky-footer/